Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +162,22 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.LOSS_FUNCTIONS[self.loss_function]
+        if self.loss_function != self.SqLoss:
+            items['Loss function'] +=  ", ε={}".format(self.epsilon)
+        items['Penalty'] = self.PENALTIES[self.penalty_type]
+        if self.penalty_type == self.ElasticNet:
+            items['Penalty'] += ": L1 : L2 = {} : {}".format(
+                                self.l1_ratio, 1.0 - self.l1_ratio)
+        items['Penalty'] = items['Penalty'] + ', α={}'.format(self.alpha)
+        items['Learning rate'] = ", η0={}".format(self.eta0)
+        if self.learning_rate != self.InvScaling:
```

Comment:
I've written this, but can't find it - perhaps I forgot to click "Comment": change `!=` to `==`. `power_t` is valid with `InvScaling`.

(To avoid unnecessary commit, you can amend this to the previous commit.)